package.json

```diff
@@ -37,11 +44,16 @@
     "grunt-nginx": "~0.2.2",
     "grunt-string-replace": "~1.3.1",
     "grunt-targethtml": "~0.2.6",
+    "husky": "^0.14.3",
```

Comment:
Is this package used anywhere? From its description it seems like it needs a configuration block in `package.json`

Reply:
Husky is a Git hook manager, it's used to add precommit hook